index.js

```diff
@@ -94,11 +94,17 @@ function createLoaderRules(languages, features, workers, publicPath) {
   const workerPaths = workers.reduce((acc, { label, output }) => Object.assign(acc, {
     [label]: `${publicPath ? `${stripTrailingSlash(publicPath)}/` : ''}${output}`,
   }), {});
+
+  const getWorkerPath = (workerPath) => {
+    if(!publicPath && window.__webpack_public_path__) {
+      return `${window.__webpack_public_path__}/${workerPath}`
```

Comment:
Fixes dynamic public path use case.
see https://webpack.js.org/concepts/output/#advanced

Reply:
If I understand this correctly, this function will be run at runtime – in which case, won't the `publicPath` on line 99 cause a `ReferenceError` in strict mode?